Manito/Manito/Screens/Detail-Wait/DetailWaitViewController.swift

```diff
@@ -49,183 +49,157 @@ final class DetailWaitViewController: BaseViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.fetchRoomData()
         self.configureDelegation()
         self.configureNavigationController()
+        self.detailWaitViewModel.fetchRoomInformation()
```

Comment:
p5;

저는 viewDidLoad 에 쓰이는 것들이 함수들이면 좋겠다고 생각해요! 
그래서 이 부분을 함수로 뺐으면 좋겠어요!
```suggestion
        self.fetchRoomInfomation()
        
        private func fetchRoomInfomation() {
        self.detailWaitViewModel.fetchRoomInformation()
        }
```

Reply:
넵! 수정했습니다
함수명을 `fetchRoomInformation`으로 하면 네트워크 통신이 이루어지는 함수같아서 네이밍을 `fetchRoomInformationAtViewModel` 이렇게 해봤는데 괜찮나요??
[7c696fe](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/462/commits/7c696fe69de7659af28fdb0fd788c73041324803)